ui/apps/platform/src/queries/summaryCounts.ts

```diff
@@ -0,0 +1,21 @@
+import { gql } from '@apollo/client';
```

Comment:
I moved this query and return type here, since it is being used by both the old and new dashboards at the same time. It seems that `src/queries` doesn't see a lot of updates though, is this the best location for shared queries?

(Alternatively, do we want to move away from GQL and just use the REST API here?)